CMakeLists.txt

```diff
@@ -29,6 +29,18 @@ if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
     include_directories(${VXL_CORE_INCLUDE_DIR})
     include_directories(${VXL_VCL_INCLUDE_DIR})
 
+    # for pyvxl
+    include_directories(${VXL_SOURCE_DIR})
+    include_directories(${VXL_SOURCE_DIR}/core)
+    include_directories(${VXL_SOURCE_DIR}/vcl)
```

Comment:
Are lines 34 and 35 needed?  Seems redundant with lines 29 and 30

Reply:
You're right, they were redundant.